src/about/person/description.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyles } from 'about/style'
+import { Name } from 'about/person/name'
+import { useDescription } from 'about/person/use-description'
+import { HalfRowItem } from 'about/components/item-responsive'
+
+export const Description = ({ person }) => <HalfRowItem>
+  <Name person={person} />
+  <Typography>{person.job}</Typography>
```

Comment:
Let's also extract the `job` and `description` parts so that our code ends up looking like this:
```js
export const Information = ({ person }) => <HalfRowItem>
    <Name person={person} />
    <Job person={person} />
    <Description person={person} />
</HalfRowItem>
```